test/external/dist/test_world.py

```diff
@@ -0,0 +1,58 @@
+from extra import dist
+from tinygrad.jit import TinyJit
+if __name__ == "__main__":
+  dist.preinit()
+
+from extra.dist import world
+from tinygrad.helpers import getenv
+from tinygrad.tensor import Tensor
+import numpy as np
+
+@TinyJit
+def send_jit(t, target_rank, cache_id=None) -> Tensor:
+  return world.send(t, target_rank, cache_id=cache_id).realize()
+
+@TinyJit
+def recv_jit(t, target_rank, cache_id=None) -> Tensor:
+  (t + 1).realize() # this does nothing but make sure that jit doesn't error
```

Comment:
So I comment this out and get `AssertionError: some input tensors not found`

Why can't the JIT see the buffers in the recv?

Reply:
because t isn't actually used in the sense that jit cares about it in `recv`, `recv` just requires the shape and dtype of t.

Reply:
I'm trying to understand if the JIT should be changed or not.

Reply:
Yea, I just realized that it could probably reuse the rawbuffer, or maybe I already tried that and ran into something, anyways, I'll try that again.

Reply:
Yea you are passing it in...it's unintuitive that it shouldn't use it. If it doesn't use it, the API should probably be changed to not take it.